### PR TITLE
Cmdct 4026 - Prior Authorization Section B 

### PR DESCRIPTION
**Pull request by britt-mo**
### Description
Section B:
2 questions, each with radio button responses and nested fields on "Yes".  
Validation: Text fields should only accept whole numbers. 
This section should be included on the Review and Submit page with statusing. 
This section should be included in the PDF.
This section should be under a feature flag with tickets 4028 and 4027. 

### Related ticket(s)

CMDCT-4026

---
### How to test
Flag on: see that after creating a mcpar, see that section B has `XII Prior Authorization`, and that section appears in the Review & Submit page and is included in the exported report.
Flag off: see that section B does not have `XII Prior Authorization`, and that section is not on the Review & Submit page, nor the exported report.

### Notes

---
### Pre-review checklist

- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
- [ ] I have performed a self-review of my code
- [ ] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [ ] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by karla-vm — APPROVED** [review]
LGTM!!

File: services/app-api/forms/mcpar.json [hunk 1]
```diff
@@ -630,6 +630,170 @@
...
+        {
+          "name": "XIII: Prior Authorization",
+          "path": "/mcpar/state-level-indicators/prior-authorization",
+          "pageType": "standard",
+          "verbiage": {
+            "intro": {
+              "section": "Section B: State-Level Indicators",
+              "subsection": "Topic XIII: Prior Authorization",
+              "spreadsheet": "B_State",
+              "alert": "<b>Beginning June 2026, Indicators B.XIII.1a-b-2a-b must be completed. Submission of this data before June 2026 is optional; if you choose not to respond prior to  June 2026, select “Not reporting data”.</b>"
```
Comment (bangbay-bluetiger) [inline services/app-api/forms/mcpar.json:643]: Nit: There's two spaces between `to  June`.

File: services/app-api/utils/formTemplates/formTemplates.ts [hunk 2]
```diff
@@ -317,3 +323,16 @@ const makePCCMTemplateModifications = (reportTemplate: ReportJson) => {
...
   programTypeQuestion.props!.disabled = true;
 };
+
+const handleTemplateForNovMcparRelease = (originalReportTemplate: any) => {
+  const reportTemplate = structuredClone(originalReportTemplate);
+  for (let route of reportTemplate.routes) {
+    if (route?.children) {
+      route.children = route.children.filter(
+        (childRoute: ReportRoute) =>
+          childRoute.path != "/mcpar/state-level-indicators/prior-authorization"
```
Comment (bangbay-bluetiger) [inline services/app-api/utils/formTemplates/formTemplates.ts:333]: Nit: Prefer strict comparison `!==`.

**Review by bangbay-bluetiger — DISMISSED** [review]
A couple of non-blockers, but looks good!

**Review by bangbay-bluetiger — APPROVED** [review]

**Review by karla-vm — APPROVED** [review]

**Pull request merged**